.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,21 @@
+# Approve PRs with "pr/auto-approve". mergify takes care of the actual merge.
+
+name: auto-approve
+on: pull_request
+
+jobs:
+  auto-approve:
+    if: >
+       (
+         contains(github.event.pull_request.labels.*.name, 'pr/auto-approve')
+         && (
+           github.event.pull_request.user.login == 'aws-cdk-automation' 
+           || github.event.pull_request.user.login == 'dependabot[bot]' 
+           || github.event.pull_request.user.login == 'dependabot-preview[bot]'
```

Comment:
Would this work or am I expecting too much from GH actions?

```ts
contains(['aws-cdk-automation', 'dependabot[bot]', 'dependabot-preview[bot]'], github.event.pull_request.user.login)
```

Reply:
Yes, [that would work](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contains). TIL